docs/component-guidance/cta.md

```diff
@@ -0,0 +1,158 @@
+
+_**Accessibility note:** this state does not need to meet colour contrast ratios. The W3C states:
+“1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following:
+Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
+Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement._
+
+![Disabled state](https://user-images.githubusercontent.com/78355810/144259095-ae1a9d02-0f5b-4817-bec6-b9005165967e.png)
+
+
+
+
+---
+
+## Accessible use of contrast
+
+Ensure that colours used meet or exceed WCAG AA standard for the size of text used, against its background colour. 
+
+
+![Disabled state](https://user-images.githubusercontent.com/78355810/177174311-056405b6-0985-411b-8a5f-1da399db4cec.png)
+
+Note: AA result is for 16px text, AAA is at least 18.66px with bold weight or 24px regular.
+
+---
+
+## Content
+
+The ```cta``` component contains a link to another location (href string).
+
+
+![Disabled state](https://user-images.githubusercontent.com/78355810/177314258-1b099114-e1eb-493a-919a-6feec39e9bef.png)
+
+
+Bi-directional text and icon position accommodates i18n (internationalisation) where some languages are read right to left.
+
+Loading message is 'false' by default but state can be changed using 'true'. The message default is "Loading..." And overwrites the CTA anonymous slot (the original CTA text) when active.
+
+Tone of voice - text should clearly describe what the action is eg. "Get a quote" and not "Click here". The the tense changes relevant to loading state, "Get a quote" becomes "Getting quote...".
+
+![Disabled state](https://user-images.githubusercontent.com/78355810/177314855-bdd83328-8c11-4dfa-b472-205107a035e7.png)
+
+Length should be short, relevant and actionable. No longer than 24 characters.
+
+---
+
+## Behaviour
+
+In addition to the different states changing, if the CTA is long (up to 24 characters) and the ```cta``` component is within a confined space, then the text may wrap, expanding the background downwards but still hugging the content and maintaining padding and space between text and icon. It should align to the middle vertically, and the icon too.
+
+Motion (such as that on the loading icon by default) should be minimal and subtle so as not to cause distraction or nausea amongst other reactions from animations.
+
+People with vestibular disorders need control over movement triggered by interactions. Refer to [WCAG 2.1 - Understanding Success Criterion 2.3.3: Animation from Interactions](https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html#:~:text=Success%20Criterion%202.3.-,3%3A%20Animation%20from%20Interactions,or%20the%20information%20being%20conveyed.) for more on this...
```

Comment:
This seems to general to be inside the cta guidance

Reply:
Removed as it was applicable to motion.